### DISCUSSION TITLE
About Some  meta data field in App router

### DISCUSSION THREAD

**Discussion by makhdum530** [vercel/next.js#63043] [App Router]
<meta name="msapplication-TileColor" content={'#E61B47'} />
                    <meta name="msapplication-TileImage" content={getIconList().metaIcon.MsIcon144.src} />
                            <meta name="geo.region" content={currentCity?.geo_region} />
                            <meta name="geo.placename" content={currentCity?.region_name} />
                            <meta name="geo.position" content={`${currentCity?.latitude};${currentCity?.longitude}`} />

how can we set this meta data using generateMetadata in app router  in next js 14

**Comment by mutasim77** [comment]
just try this:
```js
export async function generateMetadata({ params }) {
  return {
    title: '...',
    other: {
      'msapplication-TileColor': '#E61B47',
      'msapplication-TileImage': `${'images/tileimage.jpg'}`,
      'geo.region': 'IS',      // as an example, I am using the Island (in your case, it will be dynamic).
      'geo.placename': 'Lindau',
      'geo.position': '64.465585, -18.408346',
    }
  }
}
```
It will generate what you wanted.
